lib/vagrant-service-manager/command.rb

```diff
@@ -6,6 +6,15 @@ module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
     SCCLI_SERVICES = ['openshift', 'k8s']
+    KUBE_SERVICES = [
+      "etcd",
+      "kube-apiserver",
+      "kube-controller-manager",
+      "kube-scheduler",
+      "kubelet",
+      "kube-proxy",
+      "docker"
```

Comment:
One more reason to have `docker` service status account while checking for kubernetes service is, irrespective of whether docker is always going to be running for ADB / CDK box, the module to do a certain task has to be complete and not having dependency on the assumptions. This gives us modularity while porting the module / code from one component to another, making sure that module does its job irrespective of the assumption. Basically here I want all dependency listing and their accountability for checking kube service status.